pkg/minishift/provisioner/utils.go

```diff
@@ -27,19 +27,8 @@ import (
 )
 
 var (
-	engineConfigTemplateRHEL = `[Unit]
-Description=Docker Application Container Engine
-Documentation=http://docs.docker.com
-After=network.target rc-local.service
-Requires=rhel-push-plugin.socket
-Requires=docker-cleanup.timer
-
-[Service]
-Type=notify
-NotifyAccess=all
-Environment=GOTRACEBACK=crash
-Environment=DOCKER_HTTP_HOST_COMPAT=1
-Environment=PATH=/usr/libexec/docker:/usr/bin:/usr/sbin
```

Comment:
These changes are required because docker machine is now using system drop-in for configuration https://github.com/docker/machine/commit/b6e85bfadf8f02dd2a85e928b49ed02300292861